src/dc_receive_imf.rs

```diff
@@ -600,7 +600,7 @@ async fn add_parts(
                 }
             }
 
-            apply_group_changes(
+            better_msg = apply_group_changes(
```

Comment:
I'm late to the party, but:

Isn't it a problem that if `apply_group_changes()` returns `None`, the `better_msg` will be overwritten?

Like, maybe it's a `LocationStreamingEnabled` message and `better_msg` was `Some("Location streaming enabled by…"). Then `apply_group_changes()` returns `None` and `better_msg` becomes `None`.

Reply:
Made a fix: #3159